tpch_2_13_0/tpch_create_hash_partitioned_tables.ddl

```diff
@@ -0,0 +1,116 @@
+-- Sccsid:     @(#)dss.ddl	2.1.8.1
+
+CREATE TABLE nation
+(
+    n_nationkey  INTEGER not null,
+    n_name       CHAR(25) not null,
+    n_regionkey  INTEGER not null,
+    n_comment    VARCHAR(152)
+);
+
+select create_distributed_table('nation', 'n_nationkey');
```

Comment:
I think this should be `create_reference_table()` Same for `supplier `. @metdos Does that work?